App/xcbuddykit/Tests/Utils/Mocks/MockInputRequester.swift

```diff
@@ -0,0 +1,24 @@
+@testable import xcbuddykit
+
+class MockInputRequester: InputRequering {
+    
+    enum Mode {
+        case positive
+        case negative
+    }
+    
+    var mode: Mode = .positive
+    let printer: Printing = MockPrinter()
+    
+    func requestBoolUserInput(message: String) -> Bool {
```

Comment:
I'd stub the methods individually rather than having a mode property. Here you have an example of how we do it with the printer https://github.com/xcbuddy/xcbuddy/blob/master/App/xcbuddykit/Tests/Utils/Mocks/MockPrinter.swift

In this case it'd something like:

```swift
var requestBoolUserInputStub: ((String) -> Bool)?
var requestBoolUserInputCount: UInt = 0
var requestRequiredUserInputStub: ((String, String) -> String)?
var requestRequiredUserInputCount: UInt = 0
var requestOptionalUserInputStub: ((String) -> String)?
var requestOptionalUserInputCount: UInt = 0
```

Reply:
perfect, I'm going to check it then